README.md

```diff
@@ -26,14 +30,14 @@ npm start
 
 # Rewrites to S3
 
-- `GET /storage/assetPacks/:filename` => https://s3.amazonaws.com/nico.decentraland.zone/asset_packs/:filename (for the thumbnail, :id.png)
-- `GET /storage/contents/:hash` => https://s3.amazonaws.com/nico.decentraland.zone/contents/:hash
+- `GET /v1/storage/assetPacks/:filename` => https://s3.amazonaws.com/nico.decentraland.zone/asset_packs/:filename (for the thumbnail, :id.png)
```

Comment:
I think that if we're going to add `v1` here, we should add a little disclaimer up top explaining that the value comes from the .env's `API_VERSION` variable. In case we change it later. What do you think?